refs/biblio.json

```diff
@@ -596,17 +596,33 @@
         "aliasOf": "RFC6090"
     },
     "ECMA-262": {
-        "date": "June 2011",
-        "href": "http://www.ecma-international.org/publications/standards/Ecma-262.htm",
+        "title": "ECMA-262 6th Edition, The ECMAScript 2015 Language Specification",
+        "date": "June 2015",
+        "href": "http://www.ecma-international.org/ecma-262/6.0/index.html",
+        "status": "Standard",
         "publisher": "ECMA",
-        "title": "ECMAScript Language Specification, Edition 5.1"
+        "deliveredBy": [
+            "http://www.ecma-international.org/"
+        ]
     },
-    "ECMA-262-5.1": {
+    "ECMA-262-6.0": {
         "aliasOf": "ECMA-262"
     },
-    "ECMA-262-51": {
+    "ECMA-262-2015": {
         "aliasOf": "ECMA-262"
     },
+    "ECMA-262-5.1": {
+        "date": "June 2011",
+        "href": "http://www.ecma-international.org/publications/standards/Ecma-262.htm",
+        "publisher": "ECMA",
+        "title": "ECMAScript Language Specification, Edition 5.1",
+        "deliveredBy": [
```

Comment:
You might want to remove the "deliveredBy" props, which are really here to identify WG (not publishers).